assets/scripts/app/focus.js

```diff
@@ -44,18 +42,8 @@ export function onWindowFocus () {
   saveSettingsLocally()
 }
 
-function onWindowBlur () {
-  if (abortEverything) {
-    return
-  }
-
-  hideAllMenus()
-}
-
 function onVisibilityChange () {
-  if (document[system.visibilityState] === 'hidden') {
-    onWindowBlur()
```

Comment:
On further consideration -- let's go for simplicity. We made no guarantees about browser support, beyond most recent evergreen browsers and IE11, to the greatest extent possible. I'm okay with losing on `blur` and vendor-prefixed `visibilitychange` handlers on old browsers - menus hiding or not are details that are not important to support.